vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -289,6 +289,20 @@ export class DotnetInstallationValidated extends DotnetAcquisitionMessage {
     }
 }
 
+export class DotnetAcquisitionRequested extends DotnetAcquisitionMessage {
+    public readonly eventName = 'DotnetAcquisitionRequested';
+
+    constructor(public readonly version: string,
+                public readonly requestingId = '') {
+        super();
+    }
+
+    public getProperties() {
+        return {AcquisitionStartVersion : this.version,
```

Comment:
Just to clarify, these correspond to telemetry events yes?

Reply:
Yup!